src/pytest/test.py

```diff
@@ -1021,22 +1011,29 @@ def testGeoDeletion(env):
             'g1', "-0.1757,51.5156",
             'g2', "-0.1757,51.5156",
             't1', "hello")
+    env.cmd('ft.add', 'idx', 'doc3', 1.0, 'fields',
+            'g1', "-0.1757,51.5156",
+            't1', "hello")
 
     # keys are: "geo:idx/g1" and "geo:idx/g2"
-    env.assertEqual(2, env.cmd('zcard', 'geo:idx/g1'))
-    env.assertEqual(2, env.cmd('zcard', 'geo:idx/g2'))
+    env.assertEqual(3, len(env.cmd('FT.DEBUG DUMP_NUMIDX idx g1')[0]))
```

Comment:
Why it became 3?

Reply:
Since we have added above the extra doc.
Thought if we have two indices, we should have different values.

Reply:
:+1: